examples/kubernetes/generator/README.md

```diff
@@ -0,0 +1,54 @@
+Ceph Kubernetes Secret Generation
+=================================
+
+This script will generate ceph keyrings and configs as Kubernetes secrets.
+
+Sigil is required for template handling and must be installed in system PATH. Instructions can be found here: https://github.com/gliderlabs/sigil
```

Comment:
I think the most complex thing Sigil is used for is this:

https://github.com/elsonrodriguez/ceph-docker/blob/kubernetes-pr/examples/kubernetes/generator/templates/ceph/ceph.conf.tmpl

That could be cleanly replaced with some [bash](https://github.com/kubernetes/kubernetes/blob/master/cluster/aws/config-default.sh) and perhaps something like `envsubst`.

I'll tackle that once this PR is merged: https://github.com/ceph/ceph-docker/issues/338